pkgs/by-name/sw/sword/package.nix

```diff
@@ -1,32 +1,53 @@
-{ lib, stdenv, fetchurl, pkg-config, icu, clucene_core, curl }:
-
-stdenv.mkDerivation rec {
+{
+  lib,
+  stdenv,
+  fetchurl,
+  pkg-config,
+  icu,
+  clucene_core,
+  curl,
+}:
 
+stdenv.mkDerivation (finalAttrs: {
   pname = "sword";
-  version = "1.8.1";
+  version = "1.9.0";
 
   src = fetchurl {
-    url = "https://www.crosswire.org/ftpmirror/pub/sword/source/v1.8/${pname}-${version}.tar.gz";
-    sha256 = "14syphc47g6svkbg018nrsgq4z6hid1zydax243g8dx747vsi6nf";
+    url = "https://www.crosswire.org/ftpmirror/pub/sword/source/v${lib.versions.majorMinor finalAttrs.version}/sword-${finalAttrs.version}.tar.gz";
```

Comment:
IIRC it has a GitHub repo. I will look at it later

Reply:
Is it [bibletime/crosswire-sword-mirror](https://github.com/bibletime/crosswire-sword-mirror)?

Reply:
Upon close inspection that ☝️ repo does not have releases or tags, which would be nice to have to build the package…

Reply:
Hum, it looks a bit harder.

http://www.crosswire.org/sword/develop/index.jsp

They use their own SVN repo instead.

Reply:
@AndersonTorres maybe you could lend your expertise on this:

When I change the package `src` to be:
```
  src = fetchsvn {
    url = "https://crosswire.org/svn/sword/tags/sword-${lib.replaceStrings ["."] ["-"] finalAttrs.version}";
    hash = "sha256-QkCc894vrxEIUj4sWsB0XSH57SpceO2HjuncwwNCa4o=";
  };
```

and do a `nix build .#sword` I see the following error:
```
       > svn: E155000: Destination directory exists; please remove the directory or use --force to overwrite
       > svn: E155000: '/nix/store/r001qz6gcg5ks5crlksa577xwkdhg6fq-sword-sword-1-9-0-rHEAD/bindings/objc/sword' already exists
```

What could be the reason for that? Any ideas?

Reply:
Why not to use url = https://crosswire.org/svn/sword and rev=sword-1.9.0?

Reply:
Because it fails with:
> svn: E205000: Syntax error in revision argument 'sword-1.9.0'

Reply:
I will investigate it later. For now the expression is OK.